README.md

```diff
@@ -34,6 +34,11 @@ dnf install flex bison readline-devel zlib-devel openssl-devel \
   libseccomp-devel perl clang cmake postgresql postgresql-contrib protobuf-compiler \
   protobuf-devel
 ```
+* On Arch based systems, these packages are needed:
+```bash
+pacman -S base-devel libtool readline zlib flex bison libseccomp openssl clang \
```

Comment:
It's a bit more nuanced than `pacman -S`, because you might also need to synchronize the database with a package server (and perform a mandatory upgrade). However, this is not a proper place to teach arch users good manners or nuances, so this should be enough in practice.